docs/_basic/listening_modals.md

```diff
@@ -1,21 +1,23 @@
 ---
-title: Listening for view submissions
+title: Listening for view changes
 lang: en
 slug: view-submissions
 order: 12
 ---
 
 <div class="section-content">
-If a <a href="https://api.slack.com/reference/block-kit/views">view payload</a> contains any input blocks, you must listen to `view_submission` requests to receive their values. To listen to `view_submission` requests, you can use the built-in `view()` method.
 
-`view()` requires a `callback_id` of type `string` or `RegExp`.
+If a <a href="https://api.slack.com/reference/block-kit/views">view payload</a> contains any input blocks, you must listen to `view_submission` requests to receive their values.
+If the `notify_on_close` flag of a view is active, an respective event is sent when a view has been closed.
+To listen to either a `view_submission` request or `view_closed` event, you can use the built-in `view()` method.
 
-You can access the value of the input blocks by accessing the `state` object. `state` contains a values object that uses the `block_id` and unique `action_id` to store the input values.
+`view()` requires a `callback_id` of type `string` or `RegExp` or a constraint object with properties `type` and `callback_id`. 
 
 ---
 
 ##### Update views on submission
 
+You can access the value of the input blocks by accessing the `state` object. `state` contains a values object that uses the `block_id` and unique `action_id` to store the input values.
```

Comment:
I think this can be moved back to the more general section on views - unless there's a specific reason you had for moving this here!

Reply:
I have no strong objections but my reasing for moving it to the more specific subsection was, that I personally see no reason why this is important in the view_closed event case. Will these values even be available in all cases when the closed event is sent?

Reply:
It's relevant to `view_submission` still, but it looks like some similar language is now incorporated on line 12 there anyway after your edits. Nice! So I think we could move this underneath the code sample with this current section too.